scripts/airgapped/README.md

```diff
@@ -32,7 +32,10 @@ This script makes the following assumptions:
    the images for that repo
 
 ```bash
-./scripts/airgapped/get-all-images.sh releases/1.7/stable/kubeflow/bundle.yaml > images.txt
+python3 scripts/get-all-images.py \
```

Comment:
in which directory am I expected to be when running this? L15 suggests I'm in `/scripts` while this line suggests I'm in root of `bundle-kubeflow`

Reply:
Good point! All of them should be from root, for simplicity.

I've updated the line at the top and also added a line in the README
8770cb7